lib/utils.js

```diff
@@ -1,9 +1,9 @@
 const fs = require('fs')
-const Task = require('data.task')
+const Task = require('fluture')
```

Comment:
Can we do `const Future = require('fluture')` instead? Or do you like `Task` better?

Reply:
Sure, let's do it.